src/ExEss/Cms/Controller/User/CurrentController.php

```diff
@@ -0,0 +1,35 @@
+<?php declare(strict_types=1);
+
+namespace ExEss\Cms\Controller\User;
+
+use ExEss\Cms\Service\UserService;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+
+class CurrentController extends AbstractController
+{
+    private TokenStorageInterface $tokenStorage;
+
+    private UserService $userService;
+
+    public function __construct(TokenStorageInterface $tokenStorage, UserService $userService)
+    {
+        $this->tokenStorage = $tokenStorage;
+        $this->userService = $userService;
+    }
+
+    /**
+     * @Route("/Api/user/current", methods={"GET"})
```

Comment:
doesn't really block anything, but giving this a name might keep the list of routes cleaner

Reply:
left the names out since the default one is quite descriptive as well, provided proper folder/controller naming